server/data/schema.js

```diff
@@ -140,6 +144,23 @@ const addFeatureMutation = mutationWithClientMutationId({
   mutateAndGetPayload: ({ name, description, url }) => addFeature(name, description, url)
 });
 
+const createToken = mutationWithClientMutationId({
+  name: 'loginMutation',
+  inputFields: {
+    username: { type: new GraphQLNonNull(GraphQLString) },
+    password: { type: new GraphQLNonNull(GraphQLString) }
+  },
+  outputFields: {
+    jwtToken: {
+      type: new GraphQLNonNull(GraphQLString)
+    },
+    error: {
+      type: GraphQLString
```

Comment:
Make this a union, since it returns either jwtToken OR error. If it's an error no token should be returned.